Server/src/routes/authenticationRouter.ts

```diff
@@ -25,9 +25,14 @@ router.get('/resetPassword', (request: Request, response: Response, next: NextFu
 
 });
 
+router.post('/updateUserInfo', (request: Request, response: Response, next: NextFunction) => {
```

Comment:
We need to add a middleware here -  JWTAuthenticator.verifyJWT because this is a protected path. Like the /userDetails

Reply:
updated on the recent commit. thank you for pointing it out